### ISSUE TITLE
Remove the "Welcome Back" message in the lobby

### ISSUE THREAD

**Issue by not-3than** [Minehut/Meta#417]
**Platform**

In-Game

**Describe the feature**

Removing the lobby message with all the links and stuff that floods your chat when you join the lobby

**How the feature is useful**

It benefits people who are kicked or banned from servers, making it easier for them to see their ban message. Many players are confused when they get banned or kicked because all they see is the giant 10 line message sent by Minehut. It also benefits server owners getting complaints about this kind of thing.

![lobby join message](https://user-images.githubusercontent.com/67490606/101432729-3ccde400-38be-11eb-862e-622833a29135.png)

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by MiginsMC** [comment]
Or it could be made considerably smaller by making use of hyperlinks and not so many line breaks.

**Comment by DebitCardz** [comment]
Or it can only be shown after you join the proxy, or if you get kicked from a server that message gets displayed after the welcome message.

**Comment by ahoulton** [comment]
Hmm, maybe keep the message when you first join via proxy, for disconnects, get rid of them.

**Comment by ManagePasswords** [comment]
> Hmm, maybe keep the message when you first join via proxy, for disconnects, get rid of them.

I think it should be when you first join via proxy, to lobby servers because that would send the message to players if they're joining non-lobby servers which doesn't make any sense.

**Comment by not-3than** [comment]
> Or it could be made considerably smaller by making use of hyperlinks and not so many line breaks.

Honestly we were completely fine without it. Maybe just reference the current stuff like this:
```
-----------------
Welcome back {player}
Take this survey: {link}
-----------------
```
Like this pops up the first time you join the lobby for that day. And only pops up when there's some kind of promotion going on.

**Comment by honkling** [comment]
I'm gonna have to agree with the first join proxy thing. Sending only once you first join the proxy (in lobby), and then never again in your session sounds reasonable. I don't see a need to send it every time you join a lobby.

**Comment by DebitCardz** [comment]
> I'm gonna have to agree with the first join proxy thing. Sending only once you first join the proxy (in lobby), and then never again in your session sounds reasonable. I don't see a need to send it every time you join a lobby.

I agree as well but I think the best thing for both Minehut and Players is to show the kick message last (first thing you see in chat), it seems like the most reasonable compromise if Minehut won't do it per session.

**Comment by not-3than** [comment]
> I agree as well but I think the best thing for both Minehut and Players is to show the kick message last (first thing you see in chat), it seems like the most reasonable compromise if Minehut won't do it per session.

I'm not sure how practical that may be. It may be difficult, because the "Sending you to the lobby" and the kick message kind of go hand in hand. If there's a way, this would be very nice.

**Comment by honkling** [comment]
> I'm not sure how practical that may be. It may be difficult, because the "Sending you to the lobby" and the kick message kind of go hand in hand. If there's a way, this would be very nice.
Well the "Sending you to the lobby" should stay, but that 6 line media advertisement seems unnecessary to send every time we join a lobby, should be once when you connect to the proxy and that's it

**Comment by not-3than** [comment]
Update: The message has been shortened.

![lobby join message update](http://prntscr.com/whvk1n)

**Comment by BennyDoesStuff** [comment]
I'm going to go ahead and close this issue. Going forward the message will continue to be short like how it is right now (nothing more than a few lines) and will change over time as the need to promote something arises.
